services/analytics/object_details.go

```diff
@@ -159,3 +159,37 @@ func (s *Service) GetQueryExample(ctx context.Context, in *qanpb.QueryExampleReq
 	}
 	return resp, nil
 }
+
+// GetLabels gets labels in given time range for object.
+func (s *Service) GetLabels(ctx context.Context, in *qanpb.ObjectDetailsLabelsRequest) (*qanpb.ObjectDetailsLabelsReply, error) {
+	if in.PeriodStartFrom == nil {
+		return nil, fmt.Errorf("period_start_from is required:%v", in.PeriodStartFrom)
```

Comment:
```suggestion
		return nil, errors.Errorf("period_start_from is required: %v", in.PeriodStartFrom)
```

The same thing as `fmt.Errorf`, but with attached stack trace. Useful for internal (not visible to users) errors like this one.
Also, add space after `:`.